From 5aa8cccb8c6ea0dee914738f950e97ddc4b15758 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 21 Aug 2018 17:17:19 +0800
Subject: [PATCH 4497/5242] MLK-19258-6 ARM64: dts: fsl-imx8mm-evk: add two
 parameters for picophy tuning

commit  c28c29327ba349af5547efaa433a9be4dd1aed0c from
https://source.codeaurora.org/external/imx/linux-imx.git

With these two parameters tuning, it can pass USB eye diagram at evk board.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 32fe491..3820856 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -936,12 +936,16 @@
 &usbotg1 {
 	dr_mode = "otg";
 	extcon = <0>, <&typec1_ptn5110>;
+	picophy,pre-emp-curr-control = <3>;
+	picophy,dc-vol-level-adjust = <7>;
 	status = "okay";
 };
 
 &usbotg2 {
 	dr_mode = "otg";
 	extcon = <0>, <&typec2_ptn5110>;
+	picophy,pre-emp-curr-control = <3>;
+	picophy,dc-vol-level-adjust = <7>;
 	status = "okay";
 };
 
-- 
1.7.9.5

